src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/ResettableServicePass.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler;
+
+use Symfony\Bundle\FrameworkBundle\EventListener\ServiceResetSubscriber;
+use Symfony\Component\DependencyInjection\Argument\IteratorArgument;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * @author Alexander M. Turek <me@derrabus.de>
+ *
+ * @internal
+ */
+class ResettableServicePass implements CompilerPassInterface
+{
+    const RESET_TAG = 'kernel.reset';
+    const METHOD_ATTRIBUTE = 'method';
```

Comment:
the two const should be removed (for consistency at least: we don't use consts for tags in any pass yet)